pretext/Lists/StringsandLists.ptx

```diff
@@ -40,6 +52,51 @@ print("".join(wds))
   </program>
   <p>The list that you glue together (<c>wds</c> in this example) is not modified.  Also,
             you can use empty glue or multi-character strings as glue.</p>
+
+  <p>
+    <term>index Method in Lists and Strings</term>
+  </p>
+<p> One important thing to note is that even though index methods work the same in the context of strings and lists, 
+  however, they look for different things. In the context of strings, 
+  index methods look for a substring (or a character), whereas in the context of lists, index methods look for an element. 
+  The following is a great example of how these methods differ, depending on the context. Play with them to see the differences! </p>
+
+<program xml:id="ch4_sec11_p4" interactive="activecode" language="python">
+    <input>
+
+my_str = "To a great mind, nothing is little,' remarked Holmes, sententiously."
+print(my_str.index("great")) # Finds the substring "great" in my_str, expected result is 5
+print(my_str.index("m")) # Finds the character "m" in my_str, expected result is 12
+
+    </input>
+  </program>
+  <p> What happens when I split the string to a list? </p>
+
+<program xml:id="ch4_sec11_p5" interactive="activecode" language="python">
+    <input>
+my_str = "To a great mind, nothing is little,' remarked Holmes, sententiously."
+my_list = my_str.split() 
+print(my_list)
+print(my_list.index("great")) # Finds the element "great" in my_list, expected result is 3
+print(my_list.index("m")) # "m" is NOT an element in my_list, expected result is -1
+    </input>
+  </program>
+
+<p> <term> BONUS: </term> Check this example afer reading the next section (List Type Conversion Function) and spot the difference! </p>
+
+<program xml:id="ch4_sec11_p6" interactive="activecode" language="python">
+    <input>
+my_str = "To a great mind, nothing is little,' remarked Holmes, sententiously."
+
+char_by_char = list(my_str) 
+print(char_by_char)
+print(char_by_char.index("m")) # Finds the element "m" in char_by_char, expected result is 12
```

Comment:
I would emphasize that its the first occurrence of `m`, so maybe use `n` instead?

Reply:
Changed it!